Examples/Io/Root/include/ActsExamples/Io/Root/RootTrackStatesWriter.hpp

```diff
@@ -144,6 +144,11 @@ class RootTrackStatesWriter final : public WriterT<ConstTrackContainer> {
   /// truth parameter eT
   std::vector<float> m_t_eT;
 
+  /// event-unique particle identifier a.k.a barcode for hits per each surface
+  std::vector<std::vector<double>> m_particleId; // using double (8 bytes) to save uint64_t (also 8 bytes)
+  // particle barcodes for a given track state (size depends on a type of digitization, for smeared digitization is not more than 1)
+  std::vector<double> m_thisState_particleIds; 
```

Comment:
I think this can be moved to the source file

Reply:
sure - done